README.md

```diff
@@ -83,6 +83,10 @@ LLaMAV1 done
 
 LLaMAv2 done
```

Comment:
does the accuracy image reflect the latest result?

Reply:
added a new table